MekHQ/mmconf/defaultKeyBinds.xml

```diff
@@ -14,139 +14,139 @@ shift, etc).
-    <isRepeatable>1</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>scrollE</command>
-    <keyCode>68</keyCode> <!-- D -->
-    <modifier>0</modifier>
-    <isRepeatable>1</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>scrollW</command>
-    <keyCode>65</keyCode> <!-- A -->
-    <modifier>0</modifier>
-    <isRepeatable>1</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>toggleChat</command>
-    <keyCode>10</keyCode> <!-- Enter -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>turnLeft</command>
-    <keyCode>65</keyCode> <!-- Shift-A -->
-    <modifier>1</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>turnRight</command>
-    <keyCode>68</keyCode> <!-- Shift-D -->
-    <modifier>1</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>twistLeft</command>
-    <keyCode>65</keyCode> <!-- Shift-A -->
-    <modifier>1</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>twistRight</command>
-    <keyCode>68</keyCode> <!-- Shift-D -->
-    <modifier>1</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>prevUnit</command>
-    <keyCode>90</keyCode> <!-- Z -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>nextUnit</command>
-    <keyCode>67</keyCode> <!-- C -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>prevTarget</command>
-    <keyCode>81</keyCode> <!-- Shift-Q -->
-    <modifier>1</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>nextTarget</command>
-    <keyCode>69</keyCode> <!-- Shift-E -->
-    <modifier>1</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>prevWeapon</command>
-    <keyCode>81</keyCode> <!-- Q -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>nextWeapon</command>
-    <keyCode>69</keyCode> <!-- E -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>fire</command>
-    <keyCode>70</keyCode> <!-- F -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>undo</command>
-    <keyCode>8</keyCode> <!-- Backspace -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>movementEnvelope</command>
-    <keyCode>82</keyCode> <!-- R -->
-    <modifier>0</modifier>
-    <isRepeatable>false</isRepeatable>
-  </KeyBind>
-
-  <KeyBind>
-    <command>centerOnSelected</command>
-    <keyCode>32</keyCode> <!-- Spacebar -->
-    <modifier>0</modifier>
-    <isRepeatable>0</isRepeatable>
-  </KeyBind>
+             xsi:noNamespaceSchemaLocation="keyBindingSchema.xsl">
```

Comment:
I know xml can sometimes get grumpy when we change whitespaces. Just wanting to verify the changes here won't cause any issues?